src/project/internal/thumbnailcreator.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "ithumbnailcreator.h"
+
+#include "global/async/asyncable.h"
+#include "global/async/channel.h"
+
+namespace au::project {
+
+class ThumbnailCreator : public IThumbnailCreator, public muse::async::Asyncable
+{
+public:
+    ThumbnailCreator() = default;
+    void onThumbnailCreated(bool success) override;
+    muse::Ret createThumbnail(const muse::io::path_t& path) override;
+    muse::async::Channel<muse::io::path_t> captureThumbnailRequested() const override;
+
+private:
+    muse::io::path_t thumbnailPath(const muse::io::path_t& path);
```

Comment:
it can be marked as `const` method

Reply:
Sure! I don´t know why I forgot this. Thanks.